src/Scripting/Core/Hosting/CommandLine/CommandLineRunner.cs

```diff
@@ -109,14 +109,17 @@ private int RunInteractiveCore(ErrorLogger errorLogger)
             }
 
             var scriptPathOpt = sourceFiles.IsEmpty ? null : sourceFiles[0].Path;
-            var scriptOptions = GetScriptOptions(_compiler.Arguments, scriptPathOpt, _compiler.MessageProvider, diagnosticsInfos);
```

Comment:
>var [](start = 12, length = 3)

Why do you move this below? Note that diagnosticsInfos were populated with errors and reported on the next line.